frame/support/test/tests/derive_no_bound_ui/eq.stderr

```diff
@@ -4,4 +4,9 @@ error[E0277]: can't compare `Foo<T>` with `Foo<T>`
 6   | struct Foo<T: Config> {
     |        ^^^ no implementation for `Foo<T> == Foo<T>`
     |
+   ::: $RUST/core/src/cmp.rs
```

Comment:
Please revert

Reply:
hmm, I wonder why it fails locally, UI tests are using stable, right? So it should be the same 1.50 version.